sn_node/src/put_validation.rs

```diff
@@ -206,8 +206,11 @@ impl Node {
         };
         let dbc_addr = DbcAddress::from_dbc_id(&dbc_id);
 
-        debug!("validating and storing spends {:?}", dbc_addr.name());
         let key = NetworkAddress::from_dbc_address(dbc_addr).to_record_key();
+        debug!(
+            "validating and storing spends {:?} - {key:?}",
+            dbc_addr.name()
```

Comment:
I think we use `PrettyPrintRecordKey` to log keys everywhere now.
```suggestion
            "validating and storing spends {:?} - {:?}",
            dbc_addr.name(),
            PrettyPrintRecordKey::from(key.clone())
```